lib/Authy/AuthyApi.php

```diff
@@ -41,12 +41,9 @@ class AuthyApi
     public function __construct($api_key, $api_url = "https://api.authy.com")
     {
         $this->rest = new \GuzzleHttp\Client(array(
-            'base_url' => "{$api_url}/protected/json/",
-            'defaults' => array(
-                'headers'       => array('User-Agent' => $this->__getUserAgent() ),
-                'query'         => array('api_key' => $api_key),
```

Comment:
Guzzle 6 no longer uses the `defaults` subarray, to set defaults you simply pass the headers as a root element: http://guzzle.readthedocs.org/en/latest/request-options.html#headers.